tests/test_config.py

```
@@ -0,0 +1,346 @@
+    config = load_config(str(cfg))
+    assert config["include"] == ["zuul.d/**"]
+    assert config["exclude"] == ["tests/**"]
+
+
+# ---------------------------------------------------------------------------
+# CLI integration tests via subprocess
+# ---------------------------------------------------------------------------
+
+
+def _run(*args, env: dict | None = None) -> subprocess.CompletedProcess:
+    return subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)